scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -714,10 +717,21 @@
         <div class="col-2">
           {{ wtf.form_field(form.cytoband_end) }}
         </div>
+      </div>
+      <div class="row" style="margin-top:10px;">
+        <div class="col-2"></div>
         <div class="col-2">
-          {{ wtf.form_field(form.chrom_pos) }}
+          <a class="btn btn-secondary btn-sm form-control" data-bs-toggle="collapse" href="#stringcoords" aria-expanded="false" aria-controls="stringcoords">
```

Comment:
Much nicer with the collapsed field! 👍🏻 

<img width="1180" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/4fc5b320-0069-46f1-a655-d6c4aad9020f">